index.html

```diff
@@ -254,27 +254,23 @@ <h2>
         The API defined in this specification is used to retrieve the
         geographic location of a hosting device. In almost all cases, this
         information also discloses the location of the user of the device,
-        thereby potentially compromising the user's privacy. A conforming
-        implementation of this specification MUST provide a mechanism that
-        protects the user's privacy and this mechanism SHOULD ensure that no
-        location information is made available through this API without the
-        user's express permission.
+        thereby potentially compromising the user's privacy. User agents MUST
+        NOT send location information to Web sites without the express
+        permission of the user.
       </p>
       <section id="privacy_for_uas">
         <h3>
-          Privacy considerations for implementers of the Geolocation API
+          Privacy considerations for geolocation permission UI
         </h3>
         <p>
-          User agents MUST NOT send location information to Web sites without
-          the express permission of the user. User agents MUST acquire
-          permission through a user interface, unless they have prearranged
-          trust relationships with users, as described below. The user
-          interface MUST include the {{URL/host}} component of the document's
-          URL. Those permissions that are acquired through the user interface
-          and that are preserved beyond the current browsing session (i.e.
-          beyond the time when the <a>browsing context</a> is navigated to
-          another URL) MUST be revocable and user agents MUST respect revoked
-          permissions.
+          User agents MUST acquire permission through a user interface, unless
+          they have prearranged trust relationships with users, as described
+          below. The user interface MUST include the {{URL/host}} component of
```

Comment:
Is there anything special about geolocation that requires the host to be visible, or should that move to the permissions spec? Must it be the whole host, or could a UA emphasize a suffix in the case of a very long host?

Reply:
I think it should move to the permissions spec, tbh. There is nothing special about Geo. I think Geolocation was the first permission prompt to show the origin ([2010](https://marcosc.com/wp-content/uploads/2010/06/caceres_marcos_geopriv.pdf)) and why this is here... it's legacy stuff.